.github/workflows/push-jenkins-e2e-jobs.yaml

```diff
@@ -6,69 +6,11 @@ on:
       - release
 
 jobs:
-  prepare-env:
+  e2e-integrated:
```

Comment:
Can you rename whole job to 

Push ~ Run integrated e2e tests

and file to 

push-e2e-integrated-tests.yaml